### PR TITLE
Create LIT14505Alamayahu (prince)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**

### CODE REVIEW

**Comment by karljonaskarlsson** [comment]
I am almost certain that this is the same person as PRS1625Alamayy, as he seems to be the only ʿĀlamāyyahu Tewodros in the EAe. It would, however, be good to expand that record, adding, for example, a reference to this book: https://thehistorypress.co.uk/publication/the-prince-and-the-plunder/

File: new/PRS14505Alamayahu.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14505Alamayahu" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ʿAlamāyahu Tewodros</title>
```
Comment (DenisNosnitsin1970) [inline new/PRS14505Alamayahu.xml:8]: It is ʿAlamāyyahu.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

**Comment by DenisNosnitsin1970** [comment]
Anyone who wants to expend the bibliogaphy is kinldy invited to do that :) However, the book by Andrew Heavens would be not good as the only one, if at all (the author has worked  "six years as a reporter and photographer in Ethiopia and Sudan", has no academic background. there can be hardly any doubt as to how the story is presented).

**Comment by CarstenHoffmannMarburg** [comment]
> I am almost certain that this is the same person as PRS1625Alamayy, as he seems to be the only ʿĀlamāyyahu Tewodros in the EAe.

Sorry for not investigating it enough! I will update PRS1625Alamayy and delete PRS14505Alamayahu in a minute.

**Comment by DenisNosnitsin1970** [comment]
All people etc from the index of EAe have been converted to records, frequently really difficult to discover. 
:( (you have to resort to EAE spelling)

**Review by karljonaskarlsson — APPROVED** [review]

**Comment by eu-genia** [comment]
It is correct that Prince Alamayyahu is PRS1625Alamayy and LIT14505Alamayahu is an error, I will merge as is BUT FOR THE FUTURE please - if you create a record through the web app you MUST first merge and THEN delete, as only then it will be deleted in the application.

Every time you create a record through the web app the stub with the ID is stored on the server, invisible to the GitHub repository. So when you create it and do not merge in order to sync the server and the GitHub repo it still remains there - still visible on the web but unavailable in GitHub.

**Comment by eu-genia** [comment]
For a quick check to what has been inherited from EAe you can also use the spreadsheets at https://docs.google.com/spreadsheets/d/1a0LtliOVuXoRcDhgj3p5V1Z2i-M5k5_MkT6YgPgh04s/edit?usp=sharing

**Pull request merged**